R/pkg/inst/tests/testthat/test_mllib_regression.R

```diff
@@ -77,6 +77,18 @@ test_that("spark.glm and predict", {
   out <- capture.output(print(summary(model)))
   expect_true(any(grepl("Dispersion parameter for gamma family", out)))
 
+  # tweedie family
+  require(statmod)
```

Comment:
in fact, `library` is more correct here as it will fail if the package isn't installed.